src/database/database-files.h

```diff
@@ -41,3 +41,23 @@ class PlayerDatabaseFiles : public PlayerDatabase
 
 	std::string m_savedir;
 };
+
+class AuthDatabaseFiles : public AuthDatabase
+{
+public:
+	AuthDatabaseFiles(const std::string &savedir);
+	virtual ~AuthDatabaseFiles() = default;
+
+	virtual bool getAuth(const std::string &name, AuthEntry &res);
```

Comment:
this can be a const method

Reply:
I tried, and it's tricky. Any call will also call `verifyDatabase`, which will potentially create the database, flag everything as `m_initialized` etc.

Yes, I could mark any such things (the statements, too, are changed during reads until a `reset` is called) as `volatile`, but then I'd have to back-patch this into the map and player databases too; none of which, by the way, have `const` methods. Is this really that critical?

Reply:
It's not critical at all, just something I noticed.